spec/function.dd

```diff
@@ -872,15 +872,10 @@ void main()
 
 $(H4 $(LNAME2 inline-functions, Inline Functions))
 
-        $(P There is no inline keyword. The compiler makes the decision whether to
-        inline a function or not, analogously to the register keyword no
-        longer being relevant to a
-        compiler's decisions on enregistering variables.
-        (There is no register keyword either.))
-
-        $(P If a $(GLINK2 expression, FunctionLiteral) is immediately called,
```

Comment:
imagine a `pragma(inline, false)` before the Function literal and then what's said in this paragraph is wrong.